static/styles/portico/portico-signin.scss

```diff
@@ -259,11 +259,12 @@ html {
 .register-account .terms-of-service .text-error {
     position: relative;
     display: block;
-    top: -5px;
+    top: 20px;
```

Comment:
This changes only the CSS of terms of service errors right? Why is it in the same commit that changes the class of all kind of errors in registration and password reset pages? I would seperate this into two commits and explain in a bit more detail what each commits do.